### PR TITLE
I8 create custom django user model

### LINKED ISSUE #8: Create Custom Django User Model
The fields it requires
- user_id - Primary key
- username
- first_name
- last_name
- email
- password, this should be stored as a hash
- team_signup,  boolean
- has_consent,  boolean

### PR DESCRIPTION
**Pull request by Dhruv8601**
## Change Summary
**[Briefly summarise the changes that you made. Just high-level stuff]**
Created a custom user model with relevant fields
Deleted current super user
Added functions to create a user and create a super user
Created new migration
Create a django command to create super user

### Change Form
**Fill this up (NA if not available). If a certain criteria is not met, can you please give a reason.**

- [x] The pull request title has an issue number
- [x] The change works by "Smoke testing" or quick testing
- [ ] The change has tests
- [ ] The change has documentation

## Other Information
**[Is there anything in particular in the review that I should be aware of?]**
The superuser was removed and data was overwritten during the process of creating the new user model

UPDATE:
New superuser was created with same credentials as before. 

Resolve #8

### CODE REVIEW

File: server/api/urls.py [hunk 1]
```diff
@@ -19,6 +19,6 @@
 
 urlpatterns = [
     path("admin/", admin.site.urls),
-    path("api/event/", include("api.event.urls")),
-    path("api/auth/", include("api.auth.urls")),
+    # path("api/event/", include("api.event.urls")),
+    # path("api/auth/", include("api.auth.urls")),
```
Comment (ChantelleKerr) [inline server/api/urls.py:23]: We shouldnt comment out working endpoints

**Comment by ChantelleKerr** [comment]
TIP: run "flake8 ." in the server dir to see if your changes fixed the errors before pushing @makon1234

**Comment by makon1234** [comment]
for some reason, I can install flake8, I keep getting a bunch of errors

**Comment by makon1234** [comment]
I meant can't install

**Comment by ChantelleKerr** [comment]
> for some reason, I can install flake8, I keep getting a bunch of errors
what errors are you getting for flake8? 

The error you are getting on this PR is null=True on the email, in the models.py file replace it with blank=True and makemigrations again

**Comment by makon1234** [comment]
2023-06-30 14:24:10.813 [info] [Error - 2:24:10 PM] Server initialization failed.
2023-06-30 14:24:10.813 [info]   Message: Pending response rejected since connection got disposed
  Code: -32097 
2023-06-30 14:24:10.813 [info] [Error - 2:24:10 PM] The Flake8 server crashed 5 times in the last 3 minutes. The server will not be restarted. See the output for more information.
2023-06-30 14:24:10.813 [info] [Error - 2:24:10 PM] Flake8 client: couldn't create connection to server.

**Comment by Dhruv8601** [comment]
> > for some reason, I can install flake8, I keep getting a bunch of errors
> 
> what errors are you getting for flake8?
> 
> The error you are getting on this PR is null=True on the email, in the models.py file replace it with blank=True and makemigrations again

@ChantelleKerr blank=True has issues has email then becomes non nullable and requires a default value and I'm not sure what a default value for an email is. Is the email a required field? If so then null and blank can both be false, if it isn't required then null needs to be true I think.

**Review by ChantelleKerr — APPROVED** [review]
Great job, it works well! Approved :)

**Pull request merged**